js/src/components/case.tsx

```diff
@@ -47,7 +47,7 @@ const TestCase: React.FC<Prop> = (props) => {
         >
           <Row>
             <Col>{props.case.name}</Col>
-            {props.case.is_labeled_flaky && <Col span={2}>❆</Col>}
+            {props.case.is_labeled_flaky && <Col span={2}> 🥶</Col>}
```

Comment:
do you need the space before the icon?

what is this emoji? 🥶

Reply:
I think they tried to use the snowflake emoji

Reply:
i changed it to something more visible ;), the snowflake is so hard to see; the space helps with visibility too

Reply:
ah ok i'll follow up